src/app/shared/constants/graphConstants.ts

```diff
@@ -0,0 +1,14 @@
+export const GRAPH_COLUMN_NAME = 'r_name';
+export const GRAPH_COLUMN_ID = 'id';
+export const GRAPH_COLUMN_R_PRODUCT_CODE = 'r_product_code';
+export const GRAPH_ACTIVES_COUNT = 5;
+
+export const GRAPH_COLUMNS_TO_IGNORE = [
+  GRAPH_COLUMN_ID,
+  GRAPH_COLUMN_NAME,
+  GRAPH_COLUMN_R_PRODUCT_CODE
+];
+
+export const GRAPH_TYPE_COLUMN = 'column';
```

Comment:
I would prefer:

```
export enum GRAPH_TYPE = {
  COLUMN = 'columns',
  LINE = 'line',
  ...
}
```
and then you can have more strict types (and also there is no chance for typos)

Reply:
Fixed. Added enum and use that